.circleci/config.yml

```diff
@@ -25,7 +23,7 @@ jobs:
             - "81:ca:95:90:1b:25:e9:cb:e7:0c:89:41:b8:cf:7f:a0"
       - run:
           name: prepare for tests and build
-          command: ./ci_scripts/prepare_for_tests.sh && ./gradlew clean build --info -s
+          command: ./ci_scripts/prepare_for_tests.sh && ./ci_scripts/download_chromedriver.sh && ./gradlew clean build --info -s
```

Comment:
chromedriver version & download now specified in a separate file so it can be run by the cd pipeline